README.md

```diff
@@ -202,7 +202,7 @@ Creating an IPFS instance couldn't be easier, all you have to do is:
 // Create the IPFS node instance
 const node = new IPFS()
 
-node.on('start', () => {
+node.on('ready', () => {
```

Comment:
when i use 'start my code throw this error : 

```
assert.js:81
  throw new assert.AssertionError({
  ^
AssertionError: The libp2p node is not started yet
    at Node.dialByPeerInfo (/root/finder/node_modules/libp2p/src/index.js:219:5)
    at EventEmitter.self._libp2pNode.discovery.on (/root/finder/node_modules/ipfs/src/core/components/libp2p.js:37:32)
    at emitOne (events.js:96:13)
    at EventEmitter.emit (events.js:188:7)
    at Railing.discovery.on (/root/finder/node_modules/libp2p/src/index.js:70:26)
    at emitOne (events.js:96:13)
    at Railing.emit (events.js:188:7)
    at PeerInfo.create (/root/finder/node_modules/libp2p-railing/src/index.js:52:16)
    at Function.PeerInfo.create (/root/finder/node_modules/peer-info/src/index.js:115:3)
    at bootstrapers.forEach.e (/root/finder/node_modules/libp2p-railing/src/index.js:45:18)
```
with 'ready' it's works. 

i don't understand i have a callback in two cases.